scripts/executor_action.py

```diff
@@ -13,10 +13,17 @@
 
 url = 'http://localhost:12321/executor?action={action}'.format(action=action)
 
+url_activation = 'http://localhost:12321/executor?action=activate'
+
 if action == 'getStatus':
     r = requests.post(url, timeout=5)
     assert r.status_code == 200
```

Comment:
@mozarik misi Zein, kalau di response nya `isActive=false`, status_code nya masih 200 ndak ya?

Reply:
status code nya non 200 mas klo gk aktif ini udah ku test di local. Azkaban latest sih

Reply:
eh kalau non-active dia dapet non-200, berarti `assert r.status_code == 200` musti diubah.
saran saya mending ngeceknya dijadikan 1 sama ngecek `isActive == true` di if statement.

n.b. hasilnya `json.loads(r.text)['isActive'] itu bener `string` kan yah?

```suggestion
    if json.loads(r.text)['isActive'] != 'true' and r.status_code != 200:
```